src/elfpy/pricing_models/base.py

```diff
@@ -238,7 +239,7 @@ def calc_liquidity(
         # update variables by rescaling the original estimates
         bond_reserves = bond_reserves * scaling_factor
         share_reserves = share_reserves * scaling_factor
-        return (share_reserves, bond_reserves)
+        return share_reserves, bond_reserves
```

Comment:
Why remove the parenthesis? I believe python automatically groups multiple returns into a tuple, so the two statements are equivalent, although including the parenthesis is more explicit.

Reply:
i hate unnecessary stuff, so it triggerred me

Reply:
"tuples are not formed by the parentheses, but rather by use of the comma" https://docs.python.org/3/reference/expressions.html

Reply:
[ThiefMaster](https://stackoverflow.com/users/298479/thiefmaster) on [stackoverflow](https://stackoverflow.com/questions/4978567/should-a-return-statement-have-parentheses) says

> return value is the "correct" way - return is a language construct, not a function.
> If you want to return a tuple, use `return your, values, here`
> There's no need for any parenthesis (tuples are created by the , "operator", not the ())

Reply:
regular expressions!
```
grep -rs "^ **return[^(]*," .                                                                        /code/elfpy
./tests/utils_for_tests.py:    return policy_name, kwargs
./src/elfpy/utils/outputs.py:    return fig, axes, grid_spec
./src/elfpy/pricing_models/base.py:        return share_reserves, bond_reserves
./src/elfpy/pricing_models/yieldspace.py:        return lp_out, d_base, d_bonds
./src/elfpy/pricing_models/yieldspace.py:        return lp_in, d_base, d_bonds
./src/elfpy/pricing_models/yieldspace.py:        return lp_in, d_base, d_bonds
./src/elfpy/markets.py:        return market_deltas, wallet_deltas
./src/elfpy/markets.py:        return market_deltas, agent_deltas
./src/elfpy/markets.py:        return market_deltas, agent_deltas
./src/elfpy/markets.py:        return market_deltas, agent_deltas
./src/elfpy/markets.py:        return market_deltas, agent_deltas
./src/elfpy/markets.py:        return market_deltas, agent_deltas
```
I just manually changed one of the above examples, so I'm kinda cheating. but I like this look :)
also just found an errant wallet_deltas!

Reply:
Ok, I'm convinced. Just so I understand -- that grep is showing that we won't be using parenthesis to return tuples anywhere in the repo, correct?

Reply:
that was showing the existing cases where we were already using no-parenthesis return, which is higher than the alternative. i think there were 4 cases where we were using parenthesis, and 2 of them were like `(0,0)`. I've changed those all over now.